src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/TelemetryUtility.cs

```diff
@@ -208,5 +210,34 @@ internal static bool IsVsOfflineFeed(PackageSource source, string expectedVsOffl
                 return null;
             }
         });
+
+        /// <summary>
+        /// Converts a collection of timings to a json array formatted string.
+        /// Empty if the collection is null or empty.
+        /// </summary>
+        /// <param name="sourceTimings">The timings to convert.</param>
+        /// <returns>A json array of timings, returns string.empty if the collection is null or empty.</returns>
+        public static string ToJsonArrayOfTimingsInSeconds(IEnumerable<TimeSpan> sourceTimings)
```

Comment:
Unless I am missing something, I don't find tests for this method in https://github.com/NuGet/NuGet.Client/blob/dev/test/NuGet.Clients.Tests/NuGet.VisualStudio.Common.Test/Telemetry/TelemetryUtilityTests.cs

Reply:
This method was just moved, so I didn't add tests originally. 
Added tests in the latest commit.